test/passes/optimize-instructions_all-features.wast

```diff
@@ -463,6 +463,50 @@
     )
     (i32.const 1)
   )
+  (func $compare-with-consts (param $x i32)
+    ;;   (signed)x < 0   ->   pass
```

Comment:
What does `pass` mean in these comments? It would probably be helpful to write the expected output here as well.

Reply:
it means this test will successfully optimized. comment "skip" means it will be skipped